WordPress/Sniffs/Theme/UseCapabilitiesNotRolesSniff.php

```diff
@@ -0,0 +1,231 @@
+<?php
+/**
+ * WordPress Coding Standard.
+ *
+ * @package WPCS\WordPressCodingStandards
+ * @link    https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards
+ * @license https://opensource.org/licenses/MIT MIT
+ */
+
+/**
+ * User capabilities should be used not roles.
+ *
+ * @link    https://make.wordpress.org/themes/handbook/review/required/#options-and-settings
+ *
+ * @package WPCS\WordPressCodingStandards
+ *
+ * @since   0.xx.0
+ */
+class WordPress_Sniffs_Theme_UseCapabilitiesNotRolesSniff extends WordPress_AbstractFunctionParameterSniff {
+
+	/**
+	 * The group name for this group of functions.
+	 *
+	 * @since 0.xx.0
+	 *
+	 * @var string
+	 */
+	protected $group_name = 'caps_not_roles';
+
+	/**
+	 * Array of functions with position.
+	 *
+	 * The number represents the position in the function call
+	 * passed variables, here the capability is to be listed.
+	 *
+	 * @since 0.xx.0
+	 *
+	 * @var array Function name with parameter position.
+	 */
+	protected $target_functions = array(
+		'add_dashboard_page'        => 3,
+		'add_posts_page'            => 3,
+		'add_media_page'            => 3,
+		'add_pages_page'            => 3,
+		'add_comments_page'         => 3,
+		'add_theme_page'            => 3,
+		'add_plugins_page'          => 3,
+		'add_users_page'            => 3,
+		'add_management_page'       => 3,
+		'add_options_page'          => 3,
+		'add_menu_page'             => 3,
+		'add_utility_page'          => 3,
+		'add_submenu_page'          => 4,
+		'current_user_can'          => 1,
+		'author_can'                => 2,
+		'current_user_can_for_blog' => 2,
+	);
+
```

Comment:
At a later stage, this sniff should probably also check for certain class methods, like `WP_User::has_cap()`.

Might be useful to add a @todo note in the class comment about this or to open an issue for it as a reminder.